### ISSUE TITLE
queue-monitor does not update list of plans

### ISSUE THREAD

**Issue by prjemian** [bluesky/bluesky-widgets#171]
Using the `queue-monitor` on v0.0.11 and notice that the list of plans available in the editor does not change when the environment is re-opened after editing the underlying Python source code for the available plans (add a new plan, rename, ...).  Even when the `queueserver` updates its file of existing pland and devices, per the `--update-existing-plans-devices ENVIRONMENT_OPEN` option, such as:

```bash
start-re-manager \
    --startup-dir ./  \
    --update-existing-plans-devices ENVIRONMENT_OPEN \
    --zmq-publish-console ON \
    --databroker-config bdp2022
```

## Expected Behavior

Expect the list of available plans and the type of items in the argument list will be recaptured from the qserver when the environment is opened.

## Current Behavior

Presently, must quit the `queue-monitor` application and restart to get the revised list of plans.

**Comment by prjemian** [comment]
Not seeing any related unit tests (https://github.com/bluesky/bluesky-widgets/search?q=ENVIRONMENT_OPEN).

**Comment by dmgav** [comment]
@prjemian Unfortunately there are no unit tests for the control widgets. Control widgets were implemented as a quick projects with strict deadline and then I made a few passes to fix obvious bugs. If it is decided that overall design of the widgets is acceptable for production, then we will have to go over the code and write unit tests. I am also planning to change the widgets so that they use `bluesky-queueserver-api` package to communicate with the server.
